mne/export/_edf.py

```diff
@@ -10,7 +10,32 @@
 from EDFlib.edfwriter import EDFwriter  # noqa: E402
 
 
+def _try_to_set_value(header, key, value, channel_index=None):
+    """Helper function to set key/value pairs in EDF/BDF header."""
+    # all EDFLib set functions are set<X>
+    # for example "setPatientName()"
+    func_name = f'set{key}'
+    func = getattr(header, func_name)
+
+    # some setter functions are indexed by channels
+    if channel_index is None:
+        return_val = func(value)
+    else:
+        return_val = func(channel_index, value)
+
+    # a return value of 0 indicates an error
+    if return_val != 0:
+        raise RuntimeError(f"Setting {key} with {value} "
+                           f"returned an error")
+
+
 def _export_raw(fname, raw, physical_range, fmt):
+    """Export Raw objects to EDF or BDF files.
+
+    TODO:
+    - if Info stores transducer information, allow writing here.
+    - if Info stores techniciain information, allow writing here.
```

Comment:
if there are TODOs, change PR to draft PR, and change PR title to not say "MRG"

Reply:
I would say these are future improvements because currently `Info` object does not support this type of information.

Reply:
```suggestion
    TODO: if in future the Info object supports transducer or
    technician information, allow writing those here.
```